panel/__init__.py

```diff
@@ -58,6 +58,7 @@
 from . import widgets  # noqa
 from .config import __version__, config, panel_extension as extension  # noqa
 from .depends import bind, depends  # noqa
+from .esm import JSComponent, PreactComponent, ReactComponent  # noqa
```

Comment:
Are we sure we want to import these into the Panel namespace? It will have a performance cost and make the `pn` namespace even more complicated for especially beginning Panel users.

Reply:
Agreed, `panel.custom.ReactComponent` reads well.

Reply:
Done